src/views/LoanDetails/LoanDetails.test.js

```diff
@@ -485,13 +418,58 @@ describe('LoanDetails', () => {
           }
         ],
       };
-      const { getAllByText } = renderLoanProxyDetails(updatedPropsData);
+      const { getAllByText } = renderLoanDetails(updatedPropsData);
       userEvent.click(getAllByText('Close Modal')[0]);
       userEvent.click(screen.getAllByRole('button')[1]);
       expect(screen.getAllByText('Close Modal')).toBeTruthy();
     });
   });
 
+  describe('Render LoanDetails component with loanAccountActions', () => {
+    beforeEach(() => {
+      mockAccounts.mockReset();
+      mockAccounts.mockImplementation(() => true);
+    });
+    it('should display "Fees/fines incurred" value when there are multiple "Open" loanAccountActions', () => {
```

Comment:
Please follow this test format in your tests:
```
describe('when a condition is met', () => {
  it('should have some effect', () => {...});
});
```

Then test reports are much easier to read because they will be formatted like a story requirement:
`Render LoanDetails component with loanAccountActions > when there are multiple "Open" loanAccountActions > should display "Fees/fines incurred" value`
Plus you can group multiple `it` statements within one `describe` if they have the same pre-conditions